rp2-pio/examples/ws2812b/main.go

```diff
@@ -2,16 +2,23 @@ package main
 
 import (
 	"machine"
+	"strconv"
 	"time"
 
 	pio "github.com/tinygo-org/pio/rp2-pio"
 	"github.com/tinygo-org/pio/rp2-pio/piolib"
 )
 
+var ws2812Pin string
```

Comment:
leave a comment here to tell user how to set this pin via tinygo compiler, else it is confusing as to what is happening.